### ISSUE TITLE
The program stops producing alternative keys after it runs for a while

### ISSUE THREAD

**Issue by azihassan** [Scroph/Keymapper#1]
After I removed the compile-time loading of keys, this problem started happening for some unknown reason. The program would work as expected, but then after a few hours, it would simply cease to work.

I managed to narrow it down to the `uint[uint] keyMaps` associative array. Here's what the debug blocks output when the program is running as you would expect it to :

```
Pressed : 5a
    90 == 90 && 83 != 0 : true
    1180703 < 1180629
```

Here's what gets displayed when the bug occurs :

```
Pressed : 5a
    90 == 90 && 0 != 0 : false
```

And here's the conditional statement that prevents it from going any further :

``` D
if(pressed == lastKeyPressed && keyMaps.get(pressed, 0) != 0)
```

According to the manual, get :

> Looks up key; if it exists returns corresponding value else evaluates and returns defVal.

In other words, .get will return the sentinel value of 0 when the key that is pressed wasn't defined in keyMaps. The only explanation I can think of is that at some point, the keyMaps associative array gets emptied for some unknown reason. But just to make sure that I'm not barking up the wrong tree, I'll add some code to the debug block in order to display the contents of keyMaps, just  to see if it does get emptied. I'll report back if anything comes up.
